### DISCUSSION TITLE
Problem with pages?

### DISCUSSION THREAD

**Discussion by pbnj1** [community/community#30224] [Other Feature Feedback, Questions, & Ideas]
HI,

I am having an issue deploying on github pages.  Where I would normally do it, under settings and then pages, I only see a link that takes me to a different page about github pages.  Can we only have one now?  I submitted a ticket but I dont know how long it will take for someone to get back to me.

Thanks!

**Comment by radar-blip** [comment]
I think you can have 1 for every repository. 
If you post a link to what you're working on, its very possible someone would be able to help.

**Reply by pbnj1** [reply]
Actually I figured out the problem.  My assignment needs me to deploy it at heroku, not github.  The reason it must not have even given me the option is because I dont have an index.html on my root level (because it was not needed for the assignment).  Thanks for getting back so soon!

**Reply by radar-blip** [reply]
No problem. Good luck!

**Comment by jfullstackdev** [comment]
I think you are already fine deploying GitHub Pages, 

this is the latest

![image](https://user-images.githubusercontent.com/47092464/186666800-57600a3e-febd-44ef-b482-8953fd45c076.png)
